todolist/settings.py

```diff
@@ -57,17 +57,23 @@
 
 WSGI_APPLICATION = "todolist.wsgi.application"
 
+engineString = os.environ.get("ENGINE", "")
+nameString = os.environ.get("NAME", "")
+userString = os.environ.get("USER", "")
+passwordString = os.environ.get("PASSWORD", "")
+hostString = os.environ.get("HOST", "")
+portString = os.environ.get("PORT", "")
```

Comment:
add default values for host and port, engine.

Reply:
fixed